lib/matplotlib/ticker.py

```diff
@@ -823,7 +842,11 @@ def label_minor(self, labelOnlyBase):
         """
         Switch minor tick labeling on or off.
 
-        ``labelOnlyBase=True`` to turn off minor ticks.
+        Parameters
+        ----------
+        labelOnlyBase : bool, optional, default: True
+            If True, only label decades.
```

Comment:
Documentation for `labelOnlyBase` should be describing things using base, not then refer to a different term "decades".